pgxn/neon/walproposer.c

```diff
@@ -2120,3 +2133,21 @@ FormatEvents(WalProposer *wp, uint32 events)
 
 	return (char *) &return_str;
 }
+
+/*
+ * Disables core dump for the current process.
+ */
+void
+disable_core_dump()
+{
+	struct rlimit rlim;
```

Comment:
Let's skip this when building in library mode:
```
#ifndef WALPROPOSER_LIB
	return;
#endif
```

Reply:
done